wolfssl/sniffer.h

```diff
@@ -134,6 +134,39 @@ WOLFSSL_API
 SSL_SNIFFER_API int ssl_SetConnectionCtx(void* ctx);
 
 
+typedef struct SSLStats
+{
+    unsigned int sslStandardConns;
+    unsigned int sslClientAuthConns;
+    unsigned int sslResumedConns;
+    unsigned int sslEphemeralMisses;
+    unsigned int sslResumeMisses;
+    unsigned int sslCiphersUnsupported;
+    unsigned int sslKeysUnmatched;
+    unsigned int sslKeyFails;
+    unsigned int sslDecodeFails;
+    unsigned int sslAlerts;
+    unsigned int sslDecryptedBytes;
```

Comment:
Is int big enough for some of these?

Reply:
Going to long int is reasonable. Updating.